images/hubble-proto/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Copyright 2020 Authors of Cilium
 # SPDX-License-Identifier: Apache-2.0
 
-ARG GOLANG_IMAGE=docker.io/library/golang:1.15.5-alpine3.12
+ARG GOLANG_IMAGE=docker.io/library/golang:1.15.6-alpine3.12
```

Comment:
Happy to see that hubble-proto hasn't been left out!

Reply:
@kAworu Thanks to your PR #14036 :pray: